build.gradle

```diff
@@ -307,7 +306,7 @@ dependencies {
     }
     implementation 'com.github.wnameless:json-flattener:0.5.0'
     implementation 'com.flipkart.zjsonpatch:zjsonpatch:0.4.4'
-    implementation 'org.apache.kafka:kafka-clients:3.0.1'
+    implementation 'org.apache.kafka:kafka-clients:3.0.2'
```

Comment:
Can we make a variable for the kafka version so future changes don't need to update so many lines at once?